src/features/events/l10n/messageIds.ts

```diff
@@ -160,6 +160,7 @@ export default makeMessages('feat.events', {
     shiftStart: m<{ no: number }>('Start shift {no}'),
     shifts: m<{ no: number }>('{no, plural, one {1 shift} other {# shifts}}'),
     shiftsHeader: m('Shifts'),
+    showExtraSettingsButton: m('Show extra settings'),
```

Comment:
Maybe this could be "Show additional settings" instead?

Reply:
Or "more", or "all"? Somehow "extra" sounds to me like they are special somehow, but maybe that's just me.

Reply:
Agree it sounded not great when I made it!

Reply:
Maybe we don't even need the "Show"? Maybe it could just be "More settings" or "Additional settings" or something? I'll leave it to you and trust your judgement. Just wanted to throw all ideas into the mix.